src/us/mn/state/dot/tms/server/comm/onvifptz/Service.java

```diff
@@ -53,7 +54,7 @@ public abstract class Service {
 
 	/** Logger method */
 	protected void log(String s) {
-		OnvifPTZPoller.slog("PTZCommandProp:" + s);
```

Comment:
This should be left alone -- in production, the onvif log is more appropriate than stdout.

Reply:
Fixed - stdout is used in my standalone test application, forgot to replace those in production.